src/components/Table/hoc/withTableSettings/TableColumnSetup/TableColumnSetup.tsx

```diff
@@ -201,6 +201,7 @@ export const TableColumnSetup = (props: TableColumnSetupProps) => {
 
     return (
         <div className={b(null, className)}>
+            {/* It is used to handle clicks on the switcher control */}
```

Comment:
that should not work this way

we should pass handler via some context i guess to easily pass it to rendered component